Makefile

```diff
@@ -1,5 +1,5 @@
 NAME := foremanctl
-VERSION := $(shell cat VERSION)
+VERSION := $(shell git describe)
```

Comment:
this implies tags are strictly `X.Y.Z`, right, no `v` prefix, no `foremanctl-` prefix?

Reply:
That's a good point. It's also what we do with `VERSION` and I'm not sure if we can filter it out directly in `git describe`. Is it a problem to use plain `x.y.z` tags?